.npmignore

```diff
@@ -0,0 +1,3 @@
+node_modules
+*.log
+root-directory
```

Comment:
If this is for testing it should be `test/fs-root` I think.
Also, if we don't need to persist anything there we can just remove it from here and create it on-the-fly for the tests and remove afterward.

Reply:
And it is done in testing script. You can check it out here: https://github.com/metarhia/filestorage/pull/1/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R22.

Reply:
Then why do you need this directory in git and npm ignore?

Reply:
Just in case someone will run `metatests test/` and won't do the cleanup. Though they shouldn't do this, so I'll remove it.

Reply:
I'm not sure what do you mean, why would the user do the clean-up? The test that creates a temporary directory should remove it.